mkdocs.yml

```diff
@@ -57,12 +57,13 @@ nav:
       - Integrating with Hardhat: hardhat.md
   - <strong>Run a Scan Node</strong>:
       - Quickstart: scanner-quickstart.md
-      - SLA Calculation: sla.md
+      - Manage your scanner pools: scanner-pools.md
+      - Manage your profile: node-operators.md
```

Comment:
is this file missing from the PR?

Reply:
No, the ordering is different. Thanks for noticing.

Reply:
but i dont see any file called `node-operators.md` in current master or in this PR?

Reply:
Sorry, it's a section that shouldn't exist. Just removed.